docs/Meadow/Getting_Started/Deployment/index.md

```diff
@@ -1,13 +1,7 @@
 ---
 layout: Meadow
 title: Deployment
-subtitle: Guides and documentation for Meadow
+subtitle: Deploy a Meadow application
 ---
 
-# Deploy a Meadow application
-
-* [macOS](macos/)
-* [Windows](Windows/)
-* [Linux](Linux/)
-
-## [Next - Hello, World](/Meadow/Getting_Started/MCUs/F7_Feather/)
\ No newline at end of file
+[This document has been moved](../Deploying_Meadow.OS/)
```

Comment:
Did you test this locally? iirc you have to do some %20; magic thing for when the link contains a `.` for the link to work, not sure.

Reply:
You were right. Had to encode the `.` to `%2E`. Fixed that.